examples/src/main/python/mllib/streaming_linear_regression_example.py

```diff
@@ -25,13 +25,14 @@
 # $example off$
 
 from pyspark import SparkContext
-from pyspark.streaming import StreamingContext
 # $example on$
 from pyspark.mllib.linalg import Vectors
-from pyspark.mllib.regression import LabeledPoint
-from pyspark.mllib.regression import StreamingLinearRegressionWithSGD
+from pyspark.mllib.regression import (LabeledPoint,
+                                      StreamingLinearRegressionWithSGD)
```

Comment:
This does not exceed 100 line length? Up to my knowledge Spark limits it 100 (not default 80).